### DISCUSSION TITLE
Turtle judge doesn't show the picture anymore.

### DISCUSSION THREAD

**Discussion by marcothijs** [dodona-edu/feedback-and-support#46] [Questions - Vragen] [answered]
The turtle judge doesn't show the submitted result and expected result anymore as a picture (as explained and showed in the documentation). See attached printscreen to show what I see today. Can it be fixed?

<img width="956" alt="Screenshot 2024-10-07 at 20 10 20" src="https://github.com/user-attachments/assets/0227a2f3-f67d-4d05-b39d-cc4c376e063f">

Thanks in advance. 
Marco.

**Comment by BTWS2** [comment, marked as answer]
Apparently there was already an issue about this, but I forgot about it: https://github.com/dodona-edu/judge-turtle/issues/66. I'll see if I can work with Jorg's suggestions.

**Comment by marcothijs** [comment]
Thanks, will keep an eye the issue logged last year for updates.

**Reply by BTWS2** [reply]
@marcothijs, it's fixed now (https://github.com/dodona-edu/judge-turtle/issues/66).

**Comment by marcothijs** [comment]
Great! Tested and approved :-) Thanks for your support.
